themes/bootstrap3/theme.config.php

```diff
@@ -11,14 +11,14 @@
         'flex-fallback.css::lt IE 10', // flex polyfill
     ],
     'js' => [
-        'vendor/jquery.min.js',
-        'vendor/bootstrap.min.js',
-        'vendor/bootstrap-accessibility.min.js',
-        'vendor/validator.min.js',
-        'lib/form-attr-polyfill.js', // input[form] polyfill, cannot load conditionally, since we need all versions of IE
-        'lib/autocomplete.js',
-        'common.js',
-        'lightbox.js',
+        ['file' => 'vendor/jquery.min.js', 'priority' => 110],
```

Comment:
Would it make sense to add a comment here documenting the available options?

Reply:
I just added a documentation there and tended to the other changes, please have a look!

Reply:
(also i get some strange error messages again from phpunitfast, related to files i didnt change... might be caused due to problems in the dev branch so i have a hard time executing tests on my local dev machine)

Reply:
Thanks for the progress, I think we're almost done!

The errors are related to the indentation of your comments in theme.config.php; they're not aligned the way php-cs-fixer expects them to be. You should be able to run `vendor/bin/phing php-cs-fixer` to correct the problem automatically.

If you don't have time, or if you're having trouble getting it to work, let me know and I can do it for you.

In any case, once the style issues are sorted out, I'll run the full test suite one last time and then merge if nothing fails.